tests/test_validators.py

```diff
@@ -517,11 +517,15 @@ def filter(self, **kwargs):
         assert queryset.called_with == {'race_name': 'bar', 'position': 1}
 
 
+class FancyConditionModel(models.Model):
+    id = models.IntegerField(primary_key=True)
+
+
 class UniqueConstraintModel(models.Model):
     race_name = models.CharField(max_length=100)
     position = models.IntegerField()
     global_id = models.IntegerField()
-    fancy_conditions = models.IntegerField()
```

Comment:
Would you mind keeping this model as it was and add a new model to cover the new `ForeignKey` use case. Would avoid a potential fix breaking a behaviour with non-foreign key fields. Thanks

Reply:
Thank you 🙌🏻